README.md

```diff
@@ -2,6 +2,29 @@
 
 Initialize bitrise config, step template or plugin template
 
+## How to build this project 
+Project is written in [Go](https://golang.org/) language and 
+uses [godep](https://github.com/tools/godep) as dependency management tool.
+
+You can build this project using sequence of `go` commands or refer to [bitrise.yml](./bitrise.yml) file,
+which contains workflows for this project.
+
+You can run `bitrise` workflows on your local machine using [bitrise CLI](https://www.bitrise.io/cli).
+
+Before you start, make sure 
+- `$HOME/go/bin` is added to `$PATH`
```

Comment:
- `$HOME/go/bin` (or `$GOPATH/bin` if go workspace is on custom path) is added to `$PATH`